src/ApiClient/PayPlugApiClient.php

```diff
@@ -55,9 +65,15 @@ public function initialise(string $secretKey): void
         );
     }
 
-    public function getAccount(): array
+    public function getAccount(bool $refresh = false): array
     {
-        return Authentication::getAccount($this->configuration)['httpResponse'] ?? [];
+        if ($refresh) {
+            $this->cache->delete('payplug_account_' . substr($this->configuration->getToken(), 8));
+        }
+
+        return $this->cache->get('payplug_account_' . substr($this->configuration->getToken(), 8), function (): array {
```

Comment:
`'payplug_account_' . substr($this->configuration->getToken(), 8)` extraire dans une variable